### PR TITLE
Add extra fields to admin bar controller

### PR DESCRIPTION
**Pull request by adamf321**
- **What kind of change does this PR introduce?** (Bug fix, feature,
  docs update, ...)
  feature
- **What is the current behavior?** (You can also link to an open issue
  here)
- **What is the new behavior (if this is a feature change)?**
  add additional fields
- **Does this PR introduce a breaking change?** (What changes might
  users need to make in their application due to this PR?)
- **Other information**:

### CODE REVIEW

File: lib/molecules/admin-bar/admin-bar.controller.js [hunk 1]
```diff
@@ -61,6 +61,10 @@ function lnMAdminBarController($log, $rootScope, $scope, $cookies, $timeout, $el
       vm.lnLogoutUrl = response.data.logout_url;
       vm.lnEditPageUrl = response.data.edit_page_url;
       vm.lnNewItemTypes = response.data.post_types;
+      vm.lnUserNicename = angular.isDefined( response.data.nicename ) ? response.data.nicename : '';
```
Comment (mitogh) [inline lib/molecules/admin-bar/admin-bar.controller.js:64]: I think this is always defined but the values are empty on the API, no?

Comment (mitogh) [inline lib/molecules/admin-bar/admin-bar.controller.js:64]: https://github.com/moxie-lean/wp-endpoints-admin-bar/blob/a2fff6502b0fe25947ee960946addc4085a0ea16/src/AdminBarApi.php#L96-L99

Comment (adamf321) [inline lib/molecules/admin-bar/admin-bar.controller.js:64]: But if you're running an older version of the api they won't be there. This was to make it backward compatible, not a breaking change.

Comment (mitogh) [inline lib/molecules/admin-bar/admin-bar.controller.js:64]: Ok, makes sense. 👍

**Pull request merged**